h/models/annotation_metadata.py

```diff
@@ -0,0 +1,25 @@
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql as pg
+from sqlalchemy.ext.mutable import MutableDict
+
+from h.db import Base
+
+
+class AnnotationMetadata(Base):
+    __tablename__ = "annotation_metadata"
+
+    annotation_id = sa.Column(
```

Comment:
Another naming question here.

This was `annotation_pk` on the first PR because we added that to annotation.

Now this points to `annotation_slim.id` but I'm not calling it `annotation_slim_id` just `annotation_id` which could be confused with `annotation.id` (the UUID).